### ISSUE TITLE
None is not supported

### ISSUE THREAD

**Issue by parsa** [STEllAR-GROUP/phylanx#411]
Code:
```python
@Phylanx()
def f():
    a = None
f()
```
Output:
```
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-60-374e20663e1b> in <module>()
----> 1 @Phylanx()
      2 def f():
      3     a = None
      4 f()

C:/Repos/phylanx/cmake-build-debug/python/build/lib.win-amd64-3.6\phylanx\ast\transformation.py in __init__(self, f)
    550             ast.increment_lineno(tree, actual_lineno)
    551             assert len(tree.body) == 1
--> 552             self.transformation = self.targets[target](tree, kwargs)
    553             self.__src__ = self.transformation.__src__
    554             if self.debug:

C:/Repos/phylanx/cmake-build-debug/python/build/lib.win-amd64-3.6\phylanx\ast\transformation.py in __init__(self, tree, kwargs)
     54         for arg in tree.body[0].args.args:
     55             self.defs[arg.arg] = 1
---> 56         self.__src__ = self.recompile(tree)
     57 
     58     def _Num(self, a, allowreturn=False):

C:/Repos/phylanx/cmake-build-debug/python/build/lib.win-amd64-3.6\phylanx\ast\transformation.py in recompile(self, a, allowreturn)
    452                 return self.nodes[nm](self, a, allowreturn)
    453             else:
--> 454                 return self.nodes[nm](self, a)
    455         except NotImplementedError:
    456             raise Exception('unsupported AST node type: %s' % nm)

C:/Repos/phylanx/cmake-build-debug/python/build/lib.win-amd64-3.6\phylanx\ast\transformation.py in _Module(self, a, allowreturn)
    272     def _Module(self, a, allowreturn=False):
    273         args = [arg for arg in ast.iter_child_nodes(a)]
--> 274         return self.recompile(args[0])
    275 
    276     def _Return(self, a, allowreturn=False):

C:/Repos/phylanx/cmake-build-debug/python/build/lib.win-amd64-3.6\phylanx\ast\transformation.py in recompile(self, a, allowreturn)
    452                 return self.nodes[nm](self, a, allowreturn)
    453             else:
--> 454                 return self.nodes[nm](self, a)
    455         except NotImplementedError:
    456             raise Exception('unsupported AST node type: %s' % nm)

C:/Repos/phylanx/cmake-build-debug/python/build/lib.win-amd64-3.6\phylanx\ast\transformation.py in _FunctionDef(self, a, allowreturn)
    197             s += ", "
    198         if len(args) == 2:
--> 199             s += self.recompile(args[1], True).strip(' ')
    200         else:
    201             s += '%s(' % full_node_name(a, 'block')

C:/Repos/phylanx/cmake-build-debug/python/build/lib.win-amd64-3.6\phylanx\ast\transformation.py in recompile(self, a, allowreturn)
    450         try:
    451             if allowreturn:
--> 452                 return self.nodes[nm](self, a, allowreturn)
    453             else:
    454                 return self.nodes[nm](self, a)

C:/Repos/phylanx/cmake-build-debug/python/build/lib.win-amd64-3.6\phylanx\ast\transformation.py in _Assign(self, a, allowreturn)
    291             self.defs[args[0].id] = 1
    292         a = '%s' % full_node_name(args[0], args[0].id)
--> 293         s += "(" + a + ", " + self.recompile(args[1]) + ")"
    294         return s
    295 

C:/Repos/phylanx/cmake-build-debug/python/build/lib.win-amd64-3.6\phylanx\ast\transformation.py in recompile(self, a, allowreturn)
    452                 return self.nodes[nm](self, a, allowreturn)
    453             else:
--> 454                 return self.nodes[nm](self, a)
    455         except NotImplementedError:
    456             raise Exception('unsupported AST node type: %s' % nm)

KeyError: 'NameConstant'
```

**Comment by hkaiser** [comment]
@rtohid I think this can be mapped to `nil`.

**Comment by parsa** [comment]
Resolved through #422
